### DISCUSSION TITLE
Suricata 2 Zeek on 2.3.2

### DISCUSSION THREAD

**Discussion by eelmonkey** [Security-Onion-Solutions/securityonion#3259] [Unsupported Versions] [answered]
I recently moved from 1604 to 2.3.2.  During the install, i was asked if I would like to install Zeek or Suricata for Metadata.  I chose Suricata, only because it was familiar to me.  After reading a bit more, I feel like Zeek may have been the better choice.  Is there a way to change to Zeek without having to do a complete reinstall?

**Comment by eelmonkey** [comment]
I should have stated that the install was fresh, the server is a standalone production system. OS is Cent as the fresh install was done directly from the ISO I downloaded from SO. I did do the upgrade from 2.3.2 to 2.3.3 via soup and had no problems. As it was prior to the upgrade, when I run so-status, everything is OK. Since Zeek does not appear to be on the system at all, I assume that I will have to do a complete reinstall. If I do not hear from anyone over the next few days, I will probably try running the install again on the current setup (using a 2.3.3 iso) and will try choosing Zeek instead of Suricata. If it breaks everything, I will do a complete reinstall.

**Comment by TOoSmOotH** [comment, marked as answer]
You can change mdengine in the global pillar to ZEEK instead of SURICATA. In 2.3.30 Suricata is now a great alternative to Zeek for generating meta data but it comes down to personal preference.

**Reply by eelmonkey** [reply]
Thanks I will try that.
